documentation-site/pages/blog/getting-started-with-base-web/index.mdx

```diff
@@ -245,6 +245,6 @@ By default all the components have the Uber look and feel, but the powerful cust
 You can see all the code for this post in the following demo:
 
 <Demo
-  src="https://codesandbox.io/embed/password-validatorgenerator-w-base-web-ogrn3?fontsize=14"
+  src="https://codesandbox.io/embed/password-validatorgenerator-w-base-web-forked-w72nq?fontsize=14"
```

Comment:
Thank you for finding this! I hope you don't mind, I replaced your forked sandbox with my own in the suggestion below. After that, LGTM

```suggestion
  src="https://codesandbox.io/s/password-validatorgenerator-w-base-web-8qtuc"
```